vectorsearch/test_procedures/common/search-only-schedule.json

```diff
@@ -9,7 +9,15 @@
         "operation-type": "vector-search",
         "index": "{{ target_index_name | default('target_index') }}",
         "detailed-results": true,
-        "k": {{ query_k  | default(100) }},
+        {% if query_k is defined %}
+        "k": {{ query_k }},
+        {% endif %}
+        {% if query_max_distance is defined %}
+        "max_distance": {{ query_max_distance }},
+        {% endif %}
+        {% if query_min_score is defined %}
+        "min_score": {{ query_min_score }},
+        {% endif %}
         "field" : "{{ target_field_name | default('target_field') }}",
```

Comment:
what if none of them is defined or both is defined?

Reply:
Not sure if we can have the logic check in here, but we can add in OSB params validation process

Reply:
Let's call this out in README param section

Reply:
updated doc